### PR TITLE
Distribution Team

### PR DESCRIPTION
**Pull request by jkutner**
[Readable](https://github.com/buildpacks/rfcs/blob/distribution-team/text/0000-distribution-team.md)

### CODE REVIEW

File: text/0000-distribution-team.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+[motivation]: #motivation
+
+We have two repositories that are not owned by a specific sub-team.
+
+# What it is
+[what-it-is]: #what-it-is
+
+Create a sub-team in the [project governance]() called the "Distribution Team", which will own the following repos:
+
+* [CircleCI Pack Orb](https://github.com/buildpacks/pack-orb)
```
Comment (hone) [inline text/0000-distribution-team.md:32]: Why the CircleCI orb and not the Tekton examples?

File: text/0000-distribution-team.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+- *distribution* - relates to the delivery, discovery, and integration of buildpacks.
+
+# Motivation
+[motivation]: #motivation
+
+We have two repositories that are not owned by a specific sub-team.
+
+# What it is
+[what-it-is]: #what-it-is
+
```
Comment (hone) [inline text/0000-distribution-team.md:29]: Can you also add the description/charter from the [community PR](https://github.com/buildpacks/community/pull/34/files):

> The Distribution team is responsible for maintaining tools and services that support the distribution, discovery, and integration of buildpacks.

File: text/0000-distribution-team.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+# Prior Art
+[prior-art]: #prior-art
+
+- [Platform team](https://github.com/buildpacks/community/blob/main/TEAMS.md#Platform-Team)
+
+# Unresolved Questions
+[unresolved-questions]: #unresolved-questions
+
+- Who are the maintainers?
```
Comment (hone) [inline text/0000-distribution-team.md:69]: Travis + Joe?

**Review by dfreilich — COMMENTED** [review]
In general, the idea is great. I'm not sure I love the name though - distribution sounds like it is in charge of distributing the Buildpacks products (either the lifecycle, or pack), but that seems to be more the responsibility of the platform team (as it focuses on controlling platforms on which you can run the lifecycle), and doesn't seem to accurately fit the core responsibility of the new team, which will focus more on the registration-yanking paths, and not really distributing buildpacks or the lifecycle.

**Comment by jromero** [comment]
> In general, the idea is great. I'm not sure I love the name though - distribution sounds like it is in charge of distributing the Buildpacks products (either the lifecycle, or pack), but that seems to be more the responsibility of the platform team (as it focuses on controlling platforms on which you can run the lifecycle), and doesn't seem to accurately fit the core responsibility of the new team, which will focus more on the registration-yanking paths, and not really distributing buildpacks or the lifecycle.

The term distribution, like almost every other term, can have different meaning depending on context. In this context, we are talking about the project and thereby the spec (not the products). I think of it as similar terms to "OCI distribution spec". It centers around how buildpacks should be distributed. The term you seem to be describing I think of as delivery.

All that said, if there are alternative names that would further clear this potential confusion I think it's worth discussing.

**Review by hone — CHANGES_REQUESTED** [review]
I'd like to see a description of the team.

**Review by sclevine — APPROVED** [review]

**Review by nebhale — APPROVED** [review]

**Comment by jkutner** [comment]
@hone description added in 58b8f72

**Review by hone — APPROVED** [review]

**Comment by ekcasey** [comment]
Final Comment Period with merge disposition, closing on 30 September, 2020.

**Pull request merged**